### PR TITLE
[PLT-6765] Remove font setting

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Deprecation - remove font setting

#### Ticket Link
Jira ticket: [PLT-6765](https://mattermost.atlassian.net/browse/PLT-6765)

#### Checklist
- [x] Has UI changes
- [x] Includes text changes and localization file ([.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) - deleted related ID

### CODE REVIEW

File: webapp/utils/utils.jsx [hunk 1]
```diff
@@ -737,19 +737,11 @@ export function resetTheme() {
     applyTheme(Constants.THEMES.default);
 }
 
-export function applyFont(fontName) {
+export function applyOpenSansFont() {
```
Comment (jwilander) [inline webapp/utils/utils.jsx]: I don't think we need this, we can just add that class to the body tag unless I'm mistaken

Comment (hmhealey) [inline webapp/utils/utils.jsx]: We don't even need to add another class. You can just add `font-family: 'Open Sans', 'sans-serif';` to the `.app__body` class

Comment (saturninoabril) [inline webapp/utils/utils.jsx]: noted, class is now added directly to body.

**Comment by saturninoabril** [comment]
@hmhealey noted. Updated PR with Open Sans files moved to `webapp/fonts` directory and related classes to `webapp/sass/base/_typography.scss`. 
@asaadmahmood font files related to Open Sans were still huge at 13.5MB. Currently we have variants such as regular, italic, 300, 300 italic, 600, 600 italic, 700, 700 italic, 800 & 800 italic. Not sure if all of those are actively being used by the platform. Any thoughts on this? thanks!

**Comment by asaadmahmood** [comment]
@saturninoabril we can remove anything about the 600 variant.

**Comment by saturninoabril** [comment]
@asaadmahmood noted and thanks, PR updated.

**Comment by hmhealey** [comment]
@asaadmahmood Do we need the 300 and 800 variants? If we use them anywhere, could we swap them for the regular and 700 ones to save space?

**Comment by asaadmahmood** [comment]
@hmhealey I think we should keep the 300 variant, but remove the 800 one.
Basically we should keep the weights, 300, 400, 600. That's about it.

**Comment by hmhealey** [comment]
Okay, cool. As long as we're sure we need them. I didn't realize we used the other weights

**Comment by jwilander** [comment]
@saturninoabril just needs a rebase

**Comment by saturninoabril** [comment]
Done rebase

**Pull request merged**